mixer/cmd/mixs/cmd/server.go

```diff
@@ -72,6 +72,14 @@ func serverCmd(info map[string]template.Info, adapters []adapter.InfoFn, printf,
 	if err := serverCmd.PersistentFlags().MarkHidden("configIdentityAttributeDomain"); err != nil {
 		fatalf("unable to hide: %v", err)
 	}
+	serverCmd.PersistentFlags().StringVar(&sa.LivenessProbeOptions.Path, "livenessProbePath", "",
+		"Path to the file for the liveness probe.")
```

Comment:
why is there a file ? can't the package implement liveness in process directly ?

Reply:
We create an empty file and use the existence of the file as the signal of liveness (or readiness). For the choice of this design -- please ask @myidpt

Reply:
^ is there a description of how this work and why ?